cpp/src/text/subword/subword_tokenize.cu

```diff
@@ -125,6 +129,33 @@ __global__ void kernel_compute_tensor_metadata(
   }
 }
 
+struct tabulator {
+  uint32_t __device__ operator()(cudf::size_type idx) { return ((idx % 3) == 0) ? idx : 0; }
+};
+
+// this happens if there are no tokens in the input
+tokenizer_result build_empty_result(cudf::size_type size,
+                                    uint32_t max_sequence_length,
+                                    rmm::cuda_stream_view stream,
+                                    rmm::mr::device_memory_resource* mr)
+{
+  auto zero = cudf::numeric_scalar<uint32_t>(0, true, stream);
+  auto ids  = cudf::detail::sequence(size * max_sequence_length, zero, zero, stream, mr);
+  auto mask = cudf::detail::sequence(size * max_sequence_length, zero, zero, stream, mr);
+
+  auto metadata = cudf::make_numeric_column(
+    cudf::data_type{cudf::type_id::UINT32}, size * 3, cudf::mask_state::UNALLOCATED, stream, mr);
+  auto d_metadata = cudf::mutable_column_device_view::create(*metadata, stream);
+  thrust::tabulate(rmm::exec_policy(stream),
+                   d_metadata->begin<uint32_t>(),
```

Comment:
Can we just use `mutable_column_view::begin<>` instead of `mutable_column_device_view::begin<>`?